src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -1920,3 +1980,26 @@ def keyPressEvent(self, event):
                 nextFocus.setFocus(qt.Qt.OtherFocusReason)
         else:
             super().keyPressEvent(event)
+
+
+def from_lateral_percentile_range_to_central_percentile(
+    lateral_percentile_range: tuple[float, float],
+) -> float:
+    """
+    Example: if we use lateral percentile (1st, 99th) we use 98% of the percentiles. This is the central percentile
+    """
+    return 100 - (lateral_percentile_range[0] + (100 - lateral_percentile_range[1]))
+
+
+def from_central_percentile_to_lateral_percentile_range(
```

Comment:
CamelCase and private please and maybe shorter names (`saturation_to_percentiles` or `saturation_to_percentile_range`) 😅 
But IMO those functions would be best hidden in a percentile widget